### PR TITLE
Add translator for the Colorado State Legislature

### PR DESCRIPTION
**Pull request by schwartzadev**

### CODE REVIEW

File: Colorado Leg.js [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+	let sponsors = doc.querySelectorAll(`div[class='sponsor-item']>div[class='member']>div[class='member-details']>h4`);
+
+	for (var i = sponsors.length - 1; i >= 0; i--) {
+		let sponsorName = ZU.trimInternal(sponsors[i].textContent);
+		item.creators.push(ZU.cleanAuthor(sponsorName, "sponsor", false));
+	}
+	item.legislativeBody = 'Colorado General Assembly';
+
+	item.session = doc.querySelector(`div[class='bill-session']>div>div[class='field-items']>div`).textContent;
+
```
Comment (adam3smith) [inline Colorado Leg.js]: I think we'll definitely want to add a snapshot.
`item.attachments.push({title: "Snapshot", document: doc});`

File: Colorado Leg.js [hunk 2]
```diff
@@ -0,0 +1,198 @@
+{
+	"translatorID": "1a615592-77b0-4715-a509-702b66196ff1",
+	"label": "Colorado Leg",
+	"creator": "Andrew Schwartz",
+	"target": "leg.colorado.gov",
```
[the remark this answers is not in the document]
Comment (schwartzadev) [reply Colorado Leg.js]: fixed: b90ecbf22dd9bffb7f29fd4333d3dd99a31464cc

File: Colorado Leg.js [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
+}
+
+function scrape(doc) {
+	var item = new Zotero.Item("bill");
+
+	item.title = doc.querySelector(`h1[class='node__title node-title']`).textContent;
```
[the remark this answers is not in the document]
Comment (schwartzadev) [reply Colorado Leg.js]: fixed: da625c7d798e1b464d358b416b7256492172ab1d

File: Colorado Leg.js [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+
+	let billNumber = doc.querySelector(
+		`div[class='field field-name-field-bill-number field-type-text field-label-hidden']`
+	).textContent;
+	item.billNumber = ZU.trimInternal(billNumber);
```
[the remark this answers is not in the document]
Comment (schwartzadev) [reply Colorado Leg.js]: fixed (two takes): a3908fcdacb4cfb96d6157d6eab91dec516ed735 and then 85a17b47ae4de36f6dbd8d4ee9ce1267027fff1c

File: Colorado Leg.js (hunk 1 above)
Comment (schwartzadev) [reply Colorado Leg.js]: added: 5a1c24502a971cffe38c5985387529c66ca27ab1

**Comment by schwartzadev** [comment]
@adam3smith I took care of each of your requests -- I think this is good to go! Let me know if you notice anything else you'd like my help with. 👍

**Comment by schwartzadev** [comment]
Hey @adam3smith -- just a friendly reminder about this PR. Let me know if there's anything I can help out with.

**Comment by AbeJellinek** [comment]
This is great, thank you! I'm really sorry that it took so long to get merged.

**Pull request merged**